src/pages/demos/content-container-vis.hbs

```diff
@@ -0,0 +1,26 @@
+---
+title: Visualization of content containers
+layout: blank
+styles:
+  - content-container
+---
+
+<div class="mzp-l-content mzp-t-content-sm display">
+  <p class="label"><code>.mzp-t-content-sm</code> <code>max-width: 464px</code></p>
```

Comment:
464 is the old value. I think this should be 432 now. Same with the other values.

Reply:
🤦